bats/Makefile

```diff
@@ -11,9 +13,35 @@ containers:
 
 SC_EXCLUDES ?= SC1091,SC2034,SC2154,SC2155
 
+.PHONY: lint
 lint:
 	@./scripts/bats-lint.pl $(shell find tests -name '*.bats')
 	find tests -name '*.bash' | xargs shellcheck -s bash -e $(SC_EXCLUDES)
 	find tests -name '*.bats' | xargs shellcheck -s bash -e $(SC_EXCLUDES)
 	find tests -name '*.bash' | xargs shfmt -s -d
 	find tests -name '*.bats' | xargs shfmt -s -d
+
+DEPS = bin/darwin/jq bin/linux/jq
+
+# Package bats tests with bats itself and dependencies into a single tarball for distribution
+bats.tar.gz: $(DEPS)
+	git submodule update --init --recursive
+	tar cvfz bats.tar.gz --exclude '.git*' --exclude '*/test/*' --exclude '*/docs/*' -- *
```

Comment:
```suggestion
	tar cvfz "$@" --exclude-vcs --exclude '*/test/*' --exclude '*/docs/*' -- *
```

Reply:
Ok, will do. Even though this now doesn't filter all the Github workflows from the submodules:

```
$ tar tf bats.tar.gz | grep \\.github | wc -l
      18
```

But size-wise it doesn't matter.